java/available-versions

```diff
@@ -1,12 +1,10 @@
-19_36
+19.0.2_7
 18.0.2.1_1
 18.0.2_9
 18.0.1_10
 18_36
 17.0.5_8
 17.0.4.1_1
-17.0.4_8
-17.0.3_7
```

Comment:
why did many of these Java versions disappear? They still exist and can be downloaded.
We should not remove such versions then.